src/plugins/audio-stream/backend.ts

```
@@ -0,0 +1,137 @@
+      // get a de-chained single logical stream instead.
+      const ua = ctx.req.header('User-Agent') ?? '';
+      const needsDechain = /Mozilla|VLC/i.test(ua);
+
+      ctx.header('Content-Type', 'audio/ogg');
+      ctx.header('Transfer-Encoding', 'chunked');
+
+      if (!needsDechain) {
+        ctx.header('icy-name', 'Pear Desktop');
+        ctx.header('icy-url', 'https://github.com/pear-devs/pear-desktop');
+        ctx.header(
+          'icy-audio-info',
```

ESLint

[error] Replace `··········` with `········`

(prettier/prettier)